ui/src/main/webapp/app/table/table.html

```diff
@@ -13,7 +13,7 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
-<div ng-cloak layout="column" layout-align=" stretch">
+<div ng-cloak layout="column" layout-align=" stretch" style="height: 100%">
```

Comment:
It would be great if you could move all the styling into a table specific stylesheet (app/table/table.css) and then load it in via the index.html file. Just helps to keep the styling separate from the html without interfering with the global styles.css file